src/framework/audio/internal/synthesizers/fluidsynth/fluidsequencer.cpp

```diff
@@ -152,6 +152,7 @@ void FluidSequencer::appendControlSwitch(EventSequenceMap& destination, const mp
 void FluidSequencer::appendPitchBend(EventSequenceMap& destination, const mpe::NoteEvent& noteEvent,
                                      const mpe::ArticulationTypeSet& appliableTypes, const channel_t channelIdx, const int noteIdx)
 {
+    UNUSED(noteIdx);
```

Comment:
I think we can just remove this parameter completely

Reply:
OK